VerbalExpressionsUnitTests/SanatizeTests.cs

```diff
@@ -8,7 +8,6 @@ namespace VerbalExpressionsUnitTests
     public class SanatizeTests
```

Comment:
Given that it required removing the now-unsupported exception attributes, that VS reported line ending issues in some files & git reported whitespace errors, I felt that having a consistent formatting on the files prior to updating nunit was preferable to performing the formatting afterwards.